assets/app/styles/_extensions.less

```diff
@@ -0,0 +1,9 @@
+.pod-template div[hawtio-extension] {
+  position: relative;
+}
+
+.pod-template .connect-link {
+  position: absolute;
+  right: -5px;
+  top: -66px;
```

Comment:
ok... you can't assume these offsets will work. containers have variable amounts of stuff in them (some have associated builds, others don't, some have associated source commits, others don't). Additionally, these offsets don't take line wrapping into account.